components/FlexibleHeader/src/MDCFlexibleHeaderView+ShiftBehavior.h

```diff
@@ -67,6 +67,8 @@ typedef NS_ENUM(NSInteger, MDCFlexibleHeaderContentImportance) {
  @note If self.observesTrackingScrollViewScrollEvents is YES, then this property can only be
  MDCFlexibleHeaderShiftBehaviorDisabled. Attempts to set shiftBehavior to any other value if
  self.observesTrackingScrollViewScrollEvents is YES will result in an assertion being thrown.
+
+ Default: MDCFlexibleHeaderShiftBehaviorDisabled
```

Comment:
Tip: Put this before any `Note:`

Reply:
The Default is placed at the bottom of every method's documentation in the FlexibleHeader component; I'm open to the change, but it's out of scope for this PR.